program_repair.py

```diff
@@ -1,15 +1,19 @@
+fixed_code = None
+mutants_made = False
+import hedy
+
 def insert(input_string, line, column, new_string):
```

Comment:
insert this way is very expensive because split() and join are expensive. Can we rewrite the code so it splits and joins once? So split at the beginning of make_mutants, only pass the one line to insert() and then join all code back in at the end of this?